### DISCUSSION TITLE
Account is ineligible for GitHub purchases due to restrictions under U.S. economic sanctions

### DISCUSSION THREAD

**Discussion by twiborg** [community/community#141733] [GitHub Education] [answered]
### Hi everyone,

Yesterday I applied to participate in GitHub Education, I was approved, but today I received the message "Thank you for your patience during this process. After verification, we have determined that your account is not suitable for purchases on GitHub due to restrictions related to US economic sanctions. For this reason, you will not be able to continue the transaction on GitHub.
If you do not agree with this decision or would like to learn more, please follow the link https://docs.github.com/en/github/site-policy/github-and-trade-controls .
We apologize for any inconvenience." What can be done in this case and is this problem being solved at all? I have gone through numerous discussions, and those who have this problem are now participating in this program. Maybe it's worth waiting? I live in the city of Omsk and it is not written in politics that this city and, in principle, Russia does not fall under US sanctions

**Comment by twiborg** [comment, marked as answer]
You just have to wait!

**Reply by ww249** [reply]
Have you been able to solve this problem?
